examples/fungible-token/tests/workspaces.rs

```diff
@@ -0,0 +1,413 @@
+use near_primitives::types::AccountId;
+use near_primitives::views::FinalExecutionStatus;
+use near_sdk::json_types::U128;
+use near_sdk::{ONE_NEAR, ONE_YOCTO};
+use workspaces::prelude::*;
+use workspaces::{Account, Contract, DevNetwork, Network, Worker};
+
+// TODO: find/come up with a replacement for `to_yocto` from simtests
+const TEN_THOUSAND_NEAR: u128 = 10_000 * ONE_NEAR;
+const ONE_HUNDRED_NEAR: u128 = 100 * ONE_NEAR;
+const FIFTY_NEAR: u128 = 50 * ONE_NEAR;
+const TEN_NEAR: u128 = 10 * ONE_NEAR;
+
+// TODO: We actually only use Sandbox here, but it is not being exported from workspaces, so we
+// have to be vague about the type
+async fn register_user<N: Network>(
+    worker: &Worker<N>,
+    contract: &Contract,
+    account_id: &AccountId,
+) -> anyhow::Result<()> {
+    let none: Option<bool> = None;
+    let res = contract
+        .call(&worker, "storage_deposit")
+        .args_json((account_id, none))?
```

Comment:
@austinabell is there a way to instantiate `None` with an explicit generic type in one line? I have tried `None: Option<bool>`, but apparently type ascriptions are experimental.

Reply:
The problem with `()` is that it does not match the function's signature. Even if `None` and `()` do serialize the same way, as an external user reading these examples I would be confused why we are passing `()` instead of `Option<bool>`.

`Option::<bool>::None` works though, thanks!